packages/battery/battery/example/pubspec.yaml

```diff
@@ -17,11 +17,11 @@ dev_dependencies:
     sdk: flutter
   integration_test:
     path: ../../../integration_test
-  pedantic: ^1.10.0-nullsafety
+  pedantic: ^1.10.0
 
 flutter:
   uses-material-design: true
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
+  sdk: ">=2.12.0-259.9.beta <3.0.0"
```

Comment:
Out of curiosity, is this the version that we are using for all the stable null safety publishes.

Reply:
Yes, based on https://dart.dev/null-safety/migration-guide#package-version indicating that that's the version where NNBD became stable.